src/clients/notifications-client.js

```diff
@@ -35,6 +35,18 @@ class NotificationsClient {
     })
   }
 
+  readAndRedirect (id) {
+    return this.client.sendRequest(`${this.client.baseUrl}/${id}/read_and_redirect`, {
+      method: 'PATCH'
+    })
+  }
+
+  readAllAndRedirect () {
+    return this.client.sendRequest(`${this.client.baseUrl}/read_all_and_redirect`, {
+      method: 'PATCH'
+    })
+  }
```

Comment:
We return the entire notification for `PATCH /notification-service/notifications/:id`.

```
$ bima-curl -X PATCH https://api-development.bundesimmo.de/notification-service/notifications/fca984ce-9176-481c-8d04-98815e062ebe | pretty-json
{
  "data": {
    "type": "notifications",
    "id": "fca984ce-9176-481c-8d04-98815e062ebe",
    "attributes": {
      "context": "users",
      "user_id": "fc851ba935f8420824498aee739ac897",
      "app_id": "milacrm",
      "title": "Eine Immobilie wurde zurückgezogen",
      "message": "Die Immobilie unpublish-apertment in Berlin wurde zurückgezogen.",
      "link": "https://milacrm-development.bundesimmo.de/#/rem/real_estates/23276",
      "created_at": "2019-02-03T12:34:56.000Z",
      "read_at": "2019-02-25T21:48:32.000Z",
      "expires_at": null
    }
  }
}
```

Why don't we just mark the notification as read and redirect to `link` client-side, using `shark-js-sdk`?

Also, how does the actual redirection wark? Like how does `this.client.sendRequest` and the API wark together to make us end up in the right place?